README.md

```diff
@@ -54,12 +60,19 @@ A customizable Neovim plugin to run code inside the editor
   ```
 
 ## Configuration
+  #### `setup(options)`
+  Runner comes with the following defaults:
+  ```lua
```

Comment:
I know I'm a bit picky but can you please remove the indentation in this block? When you render the markdown, this block is indented whereas the others are not. Thanks.